Structure_Engine/Compute/Integrate.cs

```diff
@@ -51,12 +51,7 @@ public static partial class Compute
         {
             Dictionary<string, object> results = new Dictionary<string, object>();
 
-            BoundingBox box = new BoundingBox();
-
-            for (int i = 0; i < curves.Count; i++)
-            {
-                box += curves[i].IBounds();
-            }
+            BoundingBox box = Geometry.Query.Bounds(curves.Select(x => x.IBounds()).ToList());
```

Comment:
Have you tried using `curvers.Sum(x => x.IBounds())`? I think it should do the same thing and can avoid adding a new method?

Reply:
Yes, `Sum` only returns numbers, so no dice

Reply:
I did however now find this way:
`            return boxes.Aggregate((x, y) => x + y);
`
which ofc would work, but adding the method was also to avoid the mistake in #1657 by having a method for it, (and we have one for `List<Point>` which can also be solved in the same way, (know that two wrongs don't make it right, but still))